components/NounInfo.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import 'react-modern-drawer/dist/index.css';
+import { Noun, NounProperty } from '../server/api/types';
+import { getNounData } from '@nouns/assets';
+import { bgcolors } from '@nouns/assets/dist/image-data.json';
+
+export default function NounInfo({
+  noun,
+  nounProperties,
+  nounSrc,
+  winner,
+  owner,
+}: {
+  noun: Noun | null;
+  nounProperties: NounProperty[] | null;
+  nounSrc: string;
+  winner: string;
+  owner: string;
+}) {
+  if (!noun || !nounProperties) {
+    return;
```

Comment:
`return` === `return undefined` and `undefined` is not a proper React element. Instead, please `return null`